src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -243,146 +328,185 @@ const RelocationBox = ({
                   </div>
                 )}
                 {/* Box Name */}
-                <FormControl>
-                  <FormLabel htmlFor="name" className={styles['form-label']}>
-                    Name
-                  </FormLabel>
-                  <Input
-                    isReadOnly={status !== 'pending changes'}
-                    id="name"
-                    // type="text"
-                    value={boxHolderNameState}
-                    onChange={e => setBoxHolderNameState(e.target.value)}
-                  />
+                <form id="edit-box-form" onSubmit={handleSubmit(onSubmit)}>
+                  <FormControl isInvalid={errors?.name}>
+                    <FormLabel htmlFor="name" className={styles['form-label']}>
+                      Name
+                    </FormLabel>
+                    <Input
+                      isReadOnly={status !== 'pending changes' || !editPendingChangesState}
+                      id="name"
+                      // type="text"
```

Comment:
Do we want to keep this `type="text"` here since the Name field should be text?

Reply:
uncommented!